Projects/SvitlanaLN/ConsoleApp1_1/Program.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+
+    public static class StringConversion
+      { static int WaitInt(string name_var)
+        {
+            Console.WriteLine($"Input {name_var} int value: ");
+            string inputString;
+        label1:
+            inputString = Console.ReadLine();
+            try
+            { return int.Parse(inputString); }
+            catch
+            {
+                Console.WriteLine($"Could not convert '{inputString}' to an int. Input {name_var} int value again: ");
+                goto label1;
```

Comment:
Please change this using While or DoWhile loop

Reply:
I have implemented this method in another program 2_1_Even.cs